README.md

```diff
@@ -1,4 +1,54 @@
 # Dcrtimestamptweet
 
-> Twitter bot to timestamp a thread when @dcrtimestamptweet is mentioned.
+Twitter bot to timestamp a thread when @dcrtimestamptweet is mentioned. This bot uses IPFS as a decentralized database to store the tweet thread. 
 
+## Installation
+
+1. Make a twitter development app and generate tokens: https://developer.twitter.com/en/docs/basics/authentication/guides/access-tokens.html
+
+2. Make a .env File
+    ```
+    $ touch .env
+    ```
+
+3. Add the generated tokens
+    ```
+    TWITTER_CONSUMER_KEY=<GENERATED_API_KEY>
+    TWITTER_CONSUMER_SECRET=<GENERATED_API_KEY_SECRET>
+    TWITTER_ACCESS_TOKEN=<GENERATED_ACCESS_TOKEN>
+    TWITTER_ACCESS_TOKEN_SECRET=<GENERATED_ACCESS_TOKEN_SECRET>
+    TRACKED_WORD="@dcrtimestamptweet" // mention which will call the bot
+    ```
+
+4. Install IPFS
```

Comment:
I just figured out that Is not required to install ipfs in your machine, neither run the IPFS daemon, it is good to have because you have the http gateway to test the changes you've made. So what do you think if we remove it from the installing steps and add in another section?